README.md

```diff
@@ -587,6 +587,7 @@ Table of Contents
   * [workers.dev](https://workers.dev) - Deploy serverless code for free on Cloudflare's global network. 100,000 free requests per day with a workers.dev subdomain.
   * [zeit.co/now](https://zeit.co/now) — Serverless platform with support for multiple languages (including static sites) and single command deployment. Free tier includes SSL, 20GB bandwidth, 100 deployments.
   * [Platform9](https://platform9.com/) - Managed Kubernetes service designed for developers. Free developer accounts come with up to 3 clusters & 20 nodes cluster.
+  * [fly.io](https://fly.io/) - Fly is a platform for applications that need to run globally. It runs your code close to users and scales compute in cities where your app is busiest. Write your code, package it into a Docker image, deploy it to Fly's platform and let that do all the work to keep your app snappy. Free for side projects, $10/mo of service credit that automatically applies to any paid service. And if you ran really small virtual machines, credits will go a long way.
```

Comment:
seems to me in order to get the free tier you have to sign up for a paid service? I dont think that applies with the definition I had in mind for this list so I am leaning to no on this one unless I read it wrong?

Reply:
It's free. You don't need to add payment service. I have already deployed one service https://fly.io/dashboard/apps/fly-image-service.

![image](https://user-images.githubusercontent.com/3367820/79728302-4a7b2d00-8320-11ea-92ff-6f0c2c565d45.png)

![image](https://user-images.githubusercontent.com/3367820/79728324-56ff8580-8320-11ea-90d7-899039d61b36.png)

It says that you can use paid service under certain amount of limit per month. But you can just use only free services.